### PR TITLE
Support Python 3

### PR DESCRIPTION
**Pull request by therve**

### CODE REVIEW

File: riak_repl/tox.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 2.0
-basepython = py27
+basepython = py27,py37
 envlist = unit, integration, flake8
```
Comment (ofek) [inline riak_repl/tox.ini]:
```
[tox]
minversion = 2.0
basepython = py37
envlist =
    py{27,37}-{unit,integration}

[testenv]
platform = linux|darwin|win32
deps =
    datadog-checks-base[deps]
    -rrequirements-dev.txt
commands =
    pip install -r requirements.in
    unit: pytest -v -m "not integration"
    integration: pytest -v -m"integration"

[testenv:flake8]
skip_install = true
deps = flake8
commands = flake8 .

[flake8]
exclude = .eggs,.tox,build
max-line-length = 120
```

File: riak_repl/datadog_checks/riak_repl/riak_repl.py [hunk 2]
```diff
@@ -42,17 +44,17 @@ class RiakReplCheck(AgentCheck):
...
+        "successful_exits",
+        "error_exits",
+        "retry_exits",
+        "soft_retry_exits",
+        "busy_nodes",
+        "fullsyncs_completed",
+        "last_fullsync_duration"
     ]
 
     def __init__(self, name, init_config, agentConfig, instances=None):
```
Comment (ofek) [inline riak_repl/datadog_checks/riak_repl/riak_repl.py]: Can remove

File: riak_repl/datadog_checks/riak_repl/riak_repl.py [hunk 3]
```diff
@@ -2,6 +2,8 @@
 import requests
 import unicodedata
 
+from six import iteritems
+
 from datadog_checks.checks import AgentCheck
```
Comment (ofek) [inline riak_repl/datadog_checks/riak_repl/riak_repl.py]:
```suggestion
from datadog_checks.base import AgentCheck
```

File: riak_repl/datadog_checks/riak_repl/riak_repl.py [hunk 4]
```diff
@@ -2,6 +2,8 @@
...
 from datadog_checks.errors import CheckException
```
Comment (ofek) [inline riak_repl/datadog_checks/riak_repl/riak_repl.py]:
```suggestion
from datadog_checks.base.errors import CheckException
```

File: riak_repl/datadog_checks/riak_repl/riak_repl.py (hunk 3 above)
Comment (therve) [reply riak_repl/datadog_checks/riak_repl/riak_repl.py]: Done

File: riak_repl/datadog_checks/riak_repl/riak_repl.py (hunk 4 above)
Comment (therve) [reply riak_repl/datadog_checks/riak_repl/riak_repl.py]: Done

File: riak_repl/datadog_checks/riak_repl/riak_repl.py (hunk 2 above)
Comment (therve) [reply riak_repl/datadog_checks/riak_repl/riak_repl.py]: Done

File: riak_repl/tox.ini (hunk 1 above)
Comment (therve) [reply riak_repl/tox.ini]: Done

File: riak_repl/tox.ini [hunk 5]
```diff
@@ -1,23 +1,18 @@
 [tox]
 minversion = 2.0
-basepython = py27
-envlist = unit, integration, flake8
+basepython = py37
+envlist =
+    py{27,37}-{unit,integration}
```
Comment (ofek) [inline riak_repl/tox.ini]: Sorry, also need flake8

**Comment by zippolyte** [comment]
Let's also pin the base check dependency to `>=4.2.0` in `setup.py`

**Pull request merged**